### ISSUE TITLE
🐛 Bug Report: 500 error when deleting documents

### ISSUE THREAD

**Issue by SMARTMICROWORLD** [appwrite/appwrite#7198]
### 👟 Reproduction steps

When I delete all docs from users collection some docs left, after deleting manually I am getting server error

### 👍 Expected behavior

It should delete all selected docs in users collection

### 👎 Actual Behavior

But some are left, manually deleting causing server error

### 🎲 Appwrite version

Appwrite Cloud

### 💻 Operating system

Linux

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by Haimantika** [comment]
Hi @SMARTMICROWORLD I have informed the team about it. Will get back to you with more updates.

**Comment by stnguyen90** [comment]
Would you please provide more information including:

1. Project id
2. Document
3. Collection schemas

**Comment by SMARTMICROWORLD** [comment]
p_id: 64d487585f702b2b6ad6
c_id: 64f9d3126ea4afc7653d

users collection is relationship with contents,comments... collections but i have set 'set doc id as null' after delete, still getting server error

**Comment by SMARTMICROWORLD** [comment]
> Would you please provide more information including:
> 
> 1. Project id
> 2. Document
> 3. Collection schemas

p_id: 64d487585f702b2b6ad6
c_id: 64f9d3126ea4afc7653d

users collection is relationship with contents,comments... collections but i have set 'set doc id as null' after delete, still getting server error

**Comment by stnguyen90** [comment]
@SMARTMICROWORLD, 

> p_id: 64d487585f702b2b6ad6

I couldn't find anything with project id `64d487585f702b2b6ad6`. Are you sure that's correct?

> users collection is relationship

1. Did you make any changes to the collections after creating the collections?
2. Are you deleting the documents client-side or server-side?

**Comment by SMARTMICROWORLD** [comment]
> @SMARTMICROWORLD,
> 
> > p_id: 64d487585f702b2b6ad6
> 
> I couldn't find anything with project id `64d487585f702b2b6ad6`. Are you sure that's correct?
> 
> > users collection is relationship
> 
> 1. Did you make any changes to the collections after creating the collections?
> 2. Are you deleting the documents client-side or server-side?

Yeah, that is correct. Getting error both side client and server. Account email is - harshdeepkumarmotihari@gmail.com
Project Id - 64d487585f702b2b6ad6

**Comment by DH-555** [comment]
I think this is solved, right?
